### PR TITLE
add IPv6 routing table support; replace icon

### PR DESCRIPTION
**Pull request by fabianfrz**
this will add the IPv6 routing table display tab to Diagnostics -> General

As discussed

### CODE REVIEW

**Comment by fabianfrz** [comment]
@fichtner can you review it?

**Comment by fabianfrz** [comment]
This is what I want to have in the next stable, so I want to merge that before the @mimugmail PR.

File: net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <menu>
   <Routing cssClass="fa fa-map-signs" order="45">
-    <General VisibleName="General" cssClass="fa fa-map-signs fa-fw" url="/ui/quagga/general/index" order="1"/>
+    <General VisibleName="General" cssClass="fa fa-cogs fa-fw" url="/ui/quagga/general/index" order="1"/>
```
Comment (fichtner) [inline net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml]: Is this a single item or the multiple ones? For settings pages we use the single one

File: net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt [hunk 2]
```diff
@@ -100,21 +100,29 @@ $(document).ready(function() {
...
 
 
 });
 </script>
 
 <!-- Navigation bar -->
 <ul class="nav nav-tabs" data-tabs="tabs" id="maintabs">
-    <li class="active"><a data-toggle="tab" href="#routing">{{ lang._('Routing Table') }}</a></li>
+    <li class="active"><a data-toggle="tab" href="#routing">{{ lang._('IPv4 Routing Table') }}</a></li>
+    <li><a data-toggle="tab" href="#routing6">{{ lang._('IPv6 Routing Table') }}</a></li>
```
Comment (fichtner) [inline net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt]: Ideally, IPv4 and IPv6 as label is enough

File: net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt [hunk 3]
```diff
@@ -100,21 +100,29 @@ $(document).ready(function() {
...
 </ul>
 
 <div class="tab-content content-box tab-content">
     <div id="routing" class="tab-pane fade in active">
     </div>
+    <div id="routing6" class="tab-pane fade in">
+    </div>
```
Comment (fichtner) [inline net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt]: Can you put closing tags on the same line for both divs?

File: net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt (hunk 2 above)
Comment (fabianfrz) [inline net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt]: There may be more than routing tables some day so I want it to be clear from the beginning

File: net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt (hunk 3 above)
Comment (fabianfrz) [inline net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt]: yes

File: net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml (hunk 1 above)
Comment (fabianfrz) [inline net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml]: This is the general settings page which currently only enables zebra.

**Comment by fabianfrz** [comment]
@fichtner I have added the changes you wanted except the one I prefer not to add. General may include more than routing tables so the information may become misleading some day.

File: net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt (hunk 2 above)
Comment (fichtner) [inline net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt]: How about "IPv4 Routes" then? Shorter is better...

Comment (fabianfrz) [inline net/quagga/src/opnsense/mvc/app/views/OPNsense/Quagga/diagnosticsgeneral.volt]: I'm fine with that

**Comment by fabianfrz** [comment]
@fichtner so this is where the next release on stable will be, while we probably will merger the things of @mimugmail to master so we can have the requested testing phase. This PR does not really include new stuff as it reuses the IPv4 code.

**Comment by mimugmail** [comment]
@fabianfrz Will #157 also be part of the new stable? The new PR changes some items in the model of 157 so it would be nice if 157 will stay in master not in stable.

**Comment by fabianfrz** [comment]
@mimugmail @fichtner can manage that - should not be too hard as it is a single commit.

**Pull request merged**